Formula/stubby.rb

```diff
@@ -65,16 +68,17 @@ def plist; <<-EOS.undent
         <key>UserName</key>
         <string>root</string>
         <key>StandardErrorPath</key>
-        <string>/var/log/stubby.log</string>
+        <string>#{var}/log/stubby/stubby.log</string>
         <key>StandardOutPath</key>
-        <string>/var/log/stubby.log</string>
+        <string>#{var}/log/stubby/stubby.log</string>
       </dict>
     </plist>
     EOS
   end
 
   test do
     (testpath/"test_message.txt").write("getdnsapi.net")
-    system "#{bin}/stubby", "-C", "#{etc}/stubby.conf", "-z", "127.0.0.1:5553", "-e", "0", "-F", "message.txt"
+    output = pipe_output("#{bin}/stubby -C #{etc}/stubby.conf -z 127.0.0.1:5553 -e 0 -F test_message.txt 2>/dev/null")
```

Comment:
Will fix. Can you give a good reference formula for this kind of test - I looked around and saw it done various different ways... Neither of the formulas mentioned in the cookbook do this....